### PR TITLE
Golem ship tweaks and additions

### PR DESCRIPTION
**Pull request by RandomMarine**
golemap

Tweaks:
Redundant machines removed.
Air-leaking thrusters sealed up. Why did doors lead to them anyway?
Doors added through ship interior, better movement throughout for the slow-ass golems.
Mining equipment vendor replaced by a subtype that replaces some items with science-related ones, referencing the golem's research dept origin

Removed from equipment vendor:
Cigar (Golems can't smoke)
Jaunter (No more easily jumping to the station.)
Kinetic Accelerator (Golems can't use these)
Jetpack Upgrade. (Golems can't wear hardsuits. Let alone use this.)
Space Cash (**NO CAPITALISTIC CORRUPTION IN GLORIOUS COMMUNIST GOLEM PARADISE**)
Super Kinetic Accelerator. (Hurr)

Added in their place is:
Science goggles, filler item pretty much. (250)
Monkey cube, for food, drawing blood, or feeding slimes (300)
Toolbelt, stolen from The Liberator's roboticists. (350)
Beaker of sulphuric acid. (500)
Grey slime extract, GHETTO XENOBIO (1000)
The Liberator's Legacy, containing a set of R&D boards. (2000)

Prices are up for debate.

Additions to the ship:
All-in-one grinder.
Two fire extinguishers.
Water tanks.
Beds and bedsheets, for surgeries.
**THE ROYAL CAPE OF THE LIBERATOR**

Fixes.
"automated trading pod" airlock returned to normal.
Turfs underneath the heater/propulsion now match lavaland's atmos, so they don't spill pressure outside.

Also adds myself to admins.txt

:cl: RandomMarine
rscadd: Additional luxuries have been added to golem ships. Including new rewards. Praise be to The Liberator!
rscdel: However, jaunters are no longer easily obtained by free golems.
/:cl:

### CODE REVIEW

File: code/modules/mining/equipment_locker.dm [hunk 1]
```diff
@@ -1062,3 +1062,52 @@
...
+	..()
+	component_parts = list()
+	component_parts += new /obj/item/weapon/circuitboard/mining_equipment_vendor/golem(null)
+	component_parts += new /obj/item/weapon/stock_parts/matter_bin(null)
+	component_parts += new /obj/item/weapon/stock_parts/matter_bin(null)
+	component_parts += new /obj/item/weapon/stock_parts/matter_bin(null)
+	component_parts += new /obj/item/weapon/stock_parts/console_screen(null)
+	RefreshParts()
+
+// TODO: Three more items.
```
Comment (RandomMarine) [inline code/modules/mining/equipment_locker.dm]: Fug.

I'll remove this once we settle on ideal prices.

**Comment by coiax** [comment]
:+1: x :100: 

@KorPhaeron will probably want to tweak some of this for balance.

I might consider a second autolathe, because it sees a lot of use anyway.

**Comment by LatD** [comment]
I'd lower most of the really high mining tool prices too. If I'm a golem who mines the entire 40 minute round I can get a total of about 2500 points. Mesons and advanced scanner have to be bought so that drops the usable points to about 1200. An upgraded resonator costs 2500 and the ship even starts with one of those.

I'd change these prices for example: 
Advanced scanner 800 -> 500 (golems having no backpack and only the terrible manual scanners, if these get changed to the new low ranged automatic ones 800 is fine)
Resonator 800 -> 300 (so that new golems can also get these easily)
Silver and diamond pickaxe, these need to be cheaper but I haven't used them so no ideas for cheaper prices
Super resonator 2500 -> 800 (a huge drop but it's not THAT good anyway compared to the normal one)

All of these (aside from scanner) are items which real miners wouldn't buy anyway with their points. They can also get more points by using the station's upgraded ORM and getting tools from science.

Prices for the new items seem good. Golems could get them if they work together gathering points but pretty slow if you're trying to get them alone. Golems would still need a 2 hour round to actually get their R&D up and running with high tech.

**Comment by RandomMarine** [comment]
I agree but I didn't feel like touching the existing item values. If there's support for it I'll go ahead and change that.

**Comment by octareenroon91** [comment]
Please add backpacks and/or satchels.

**Comment by coiax** [comment]
yeah, just use science backpacks and satchels, but make them vaguely expensive. After all, they should be considered a status symbol of sorts.

**Comment by RandomMarine** [comment]
Establish trade with the station if you want bags, or advance enough to make BoHs. Wear a bedsheet if you want status until then.

**Comment by coiax** [comment]
> advance enough to make Bag of Holdings

Heh, cute. @LatD here's a puzzle for you, can golems do R&D with only the mining dispenser equipment as external inputs (obviously also including proto and imprinter)

**Comment by coiax** [comment]
Also, you'll need to include a beaker of acid as an item that you can spawn. Can't do no R&D without acid. Unless we make watcher corpses full of sulphuric acid that we can extract somehow.

**Comment by LatD** [comment]
@coiax A sheet of plasma and autolathe can get you to maxed out levels easily! They have everything my guide needs. It'll take a long time though..

**Comment by coiax** [comment]
@LatD you'd better get on making a golem focused guide then ;)

**Comment by RandomMarine** [comment]
Didn't take too long for me to max everything out in playtesting.

Anyway, got a few small fixes I'll push later, please don't merge until then.

**Comment by coiax** [comment]
Unless it's an urgent fix, as a general rule, PRs aren't merged until they hit 24 hours old anyway. Also you've still got merge conflicts.

**Comment by RandomMarine** [comment]
Git is chinese to me and the wiki's unhelpful on the matter, how does one go about dealing with that?

**Comment by KorPhaeron** [comment]
golems aint communist they're libertarians

**Pull request merged**